plot_matrix.py

```diff
@@ -3,34 +3,66 @@
 import numpy as np
 import csv
 import argparse
+import collections
 import matplotlib
 matplotlib.use('PS')
```

Comment:
This section should be for the imports only. Put code in the main or in a function.